app/src/Form/Constraint/UrlResolverConstraintValidator.php

```diff
@@ -15,6 +15,10 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
+        if ($value === null) {
```

Comment:
wouldn't using an if (empty($value)) on the previous if check work better?

Reply:
`empty()` here would allow all values that cast to `false` like integers, floats and booleans also. I specifically want to only handle an empty string as well as the `null`   value. Even the `null` value is a concession to the way the form is handled internally where instead of an empty string a `null` value is passed...

Reply:
oh, I see now. It would have helped if I actually had read the description of the pr.